Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "v0.14.1"}
-PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "add_list_cloud_blobs"}
+PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "inc-page-count"}
```

Comment:
v0.0.8 is all you need to upgrade to for batch uploads. I think the page count bug fix can wait, but if you need it now feel free to release a new version of pipeline infrastructure.

Reply:
released and updated to v0.0.9